cetmix_tower_server/tests/test_command.py

```diff
@@ -837,3 +844,27 @@ def test_execute_python_code(self):
             command_result["error"],
             "Error in command result must be set to None",
         )
+
+    def test_execute_python_code_with_secret(self):
```

Comment:
We need to check if secret key is replaced in the command output. I didn't manage to find such a flow in this test.

Reply:
Thanks, I'll fix it